src/bokeh/models/plots.pyi

```diff
@@ -67,6 +68,8 @@ class Plot(LayoutDOM, BackgroundFill, BackgroundHatch, BorderFill, BorderHatch,
 
     extra_y_scales: dict[str, Scale] = ...
 
+    window_axis: WindowAxis = ...
```

Comment:
Given that the property is `Nullable(WindowAxis)`, then:
```suggestion
    window_axis: WindowAxis | None = ...
```
Though maybe it would be better to have `"x" | "y" | "none"` enum instead, especially if `"both"` would be implemented later (if it makes sense).

Reply:
"both" will explicitly never make sense, which is why I went with this, since None is more idiomatic Python. OTOH it seems unlikely anyone will really ever need to set this value since it is the default, so I will make it an enum.

Reply:
[49809a9](https://github.com/bokeh/bokeh/pull/14353/commits/49809a9930c9159e67331597408347efe44756e1)